docs/source/blog/release-120.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Announcing Label Studio version 1.2.0 with Webhooks!
+type: blog
+image: /images/release-120/
```

Comment:
I still need to add an image. @niklub any suggestions?

Reply:
Maybe something like this
![image](https://user-images.githubusercontent.com/6087484/129720362-52c1c712-3dcf-4362-bf54-528a15274d05.png)
we can take a screenshot of webhook settings, them gotta lot of blank space on the right - can use it to sketch what you can achieve at the end